Formula/libxml++.rb

```diff
@@ -3,6 +3,7 @@ class Libxmlxx < Formula
   homepage "https://libxmlplusplus.sourceforge.io/"
   url "https://download.gnome.org/sources/libxml++/2.42/libxml++-2.42.0.tar.xz"
   sha256 "3d032aede98a033eb5e815b4bfa9fa7b4e745268e6fd1ce8b1d0f70bcaf4736d"
+  license "LGPL-2.0-or-later"
```

Comment:
I think this should be `LGPL-2.1-only`, according to https://gitlab.gnome.org/GNOME/libxmlplusplus/-/blob/master/COPYING

Reply:
And their website states: Libxml++ is released under the LGPL version 2 or above

Ref: http://libxmlplusplus.sourceforge.net

Reply:
@chenrui333 thoughts on this one?

Reply:
I'd say we should ask upstream. It seems like they're not sure either.

Reply:
yeah, let's ask in the upstream. We can also even tag the maintainer if they are also on github. :)